### PR TITLE
TELCODOCS-1175: Remove remediation content from docs.openshift.com

### PR DESCRIPTION
**Pull request by ogradyp**
https://issues.redhat.com/browse/TELCODOCS-1175 Remove remediation content from docs.openshift.com

Applies to OpenShift version : 4.13 +

Preview: [Remediating, fencing, and maintaining nodes](https://57953--docspreview.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-remediating-fencing-maintaining-rhwa.html)

ID review completed by @danmacpherson 
Peer review completed by @mjpytlak

Thank you.

### CODE REVIEW

**Review by danmacpherson — COMMENTED** [review]
Just two pieces of feedback. Thanks, Padraig!

File: nodes/nodes/nodes-remediating-fencing-maintaining-rhwa.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="nodes-remediating-fencing-maintaining-rhwa"]
+= Remediating, fencing, and maintaining nodes
+include::_attributes/common-attributes.adoc[]
+:context: nodes-remediating-fencing-maintaining-rhwa
+
+toc::[]
+
+When node-level failures, such as the kernel hangs or network interface controllers (NICs) fail, the work required from the cluster does not decrease, and workloads from affected nodes need to be restarted somewhere. Failures affecting these workloads risk data loss, corruption, or both. It is important to ensure that the node reaches a safe state, known as `fencing` before initiating recovery of the workload, known as `remediation` and ideally, recovery of the node also.
```
[the remark this answers is not in the document]
Comment (ogradyp) [reply nodes/nodes/nodes-remediating-fencing-maintaining-rhwa.adoc]: Thanks for your feedback @danmacpherson . I have applied the fixes.

**Review by mjpytlak — COMMENTED** [review]
@ogradyp a few items for your consideration. Thanks.

File: virt/upgrading-virt.adoc [hunk 2]
```diff
@@ -10,7 +10,7 @@ Learn how Operator Lifecycle Manager (OLM) delivers z-stream and minor version u
 
 [NOTE]
 ====
-* The Node Maintenance Operator (NMO) is no longer shipped with {VirtProductName}. You can xref:../nodes/nodes/ecosystems/eco-node-maintenance-operator.adoc#node-maintenance-operator[install the NMO] from the *OperatorHub* in the {product-title} web console, or by using the OpenShift CLI (`oc`).
+* The Node Maintenance Operator (NMO) is no longer shipped with {VirtProductName}. You can *install the NMO* from the *OperatorHub* in the {product-title} web console, or by using the OpenShift CLI (`oc`).  For more information on remediation, fencing, and maintaining nodes, see the *Workload Availability for Red Hat OpenShift* documentation.
```
[the remark this answers is not in the document]
Comment (ogradyp) [reply virt/upgrading-virt.adoc:13]: Thanks for your feedback @mjpytlak . I have applied the link as you suggest.

**Review by kelbrown20 — COMMENTED** [review]
Great work! LGTM
Merging

**Comment by ogradyp** [comment]
Thank you for merging @kelbrown20

**Comment by ogradyp** [comment]
The content of this PR was  merged  in 4.13. I now also need it cherry-picked to 4.12. Many Thanks.

**Pull request merged**